### ISSUE TITLE
[Programmatic Access - Logic apps - Settings Tab]: Visible name and accessible name are different for the switch and spin control.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6096]
### Describe the issue 

User Experience:

Screen reader-dependent users will find it difficult to understand the purpose of the controls if the visible name and Accessible name (pro-grammatically defined name) are different and will be misguided to get the right information of the control. Hence will fail to perform any action related to that particular control present on the page.

Note: User credentials should NOT be included in the bug.

Pre-requisites:

- Make sure dependencies such as Azure Logic Apps (Standard), Azure Functions, C# are installed. 
- Navigate to Azure control present on the right side and select it. 
- Navigate to LogicApp icon present under workspace section, click on tat and select 'Create New Project'. 
- Navigate select stateful workflow, then enter the name of the workflow. 
- Once the workflow has been created then enter the name of the created workflow in edit field present on the top and press Enter. 
- Navigate to 'Open in current window' and select it. 
- A dialog will appear, select 'Yes', now the workflow will open in new window. 

Repro Steps:

- Open Visual Studio app. 
- Navigate to any workflow and right click on it, then select 'open designer'. 
- Navigate all the controls of the designer screen. 
- Now add "HTTP" trigger and select it from designer screen after adding it. 
- New dialog appears, navigate all the controls of 'Settings' tab. 
- Verify if visible name and accessible name are different for the switch and spin control. 

Actual Result:

Issue: Visible name and accessible name (pro-grammatically defined name) is different for the switch and spin control under "Settings" tab.

Observation: Name is defined for the both controls as "Concurrency control" but visual labels are "Limit" and Degree of parallelism". Screen reader JAWS announces the same information which is defined in Name.

Expected result:

Solution: Visual name and aria-label should be same for the switch and spin control under "Settings" tab.

Example: Name should be defined for switch as "Limit" and for spin control as "Degree of parallelism".

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*
